packages/cms/scripts/deploy.js

```diff
@@ -0,0 +1,52 @@
+const axios = require('axios').default;
+require('dotenv').config();
+
+async function main() {
+  try {
+    const { data } = await axios.post(
+      'https://api.digitalocean.com/v2/apps/21030157-a72b-40d2-afa6-0b498c63144d/deployments',
+      {
+        force_build: true,
+      },
+      {
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: `Bearer ${process.env.CMS_DEPLOY_KEY}`,
```

Comment:
Careful with this. Would plan to get this from something else long term like a vault or secure key value store

Reply:
I have heard about these in passing. Setting up the github action secrets mentioned using libsodium. so something along those lines? Good callout for the future and would love some help when we get to that point.